### ISSUE TITLE
Enable polling channels to be triggered manually

### ISSUE THREAD

**Issue by rcrichton** [jembi/openhim-core-js#616]

**Comment by hnnesv** [comment]
I think this already works by doing a GET request on that channel (in console you still have to pick a URL for the channel; making this possible).

Maybe this should be a console issue? Add a button that allows the user to trigger the channel?

**Comment by rcrichton** [comment]
Yeah it's probably a console issue rather, I just wasn't sure if we needed some core edits.
